sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveImplicitStringTypes.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.apache.spark.sql.catalyst.analysis
+
+import org.apache.spark.sql.catalyst.expressions.{Cast, Literal}
+import org.apache.spark.sql.catalyst.plans.logical.{AddColumns, AlterColumn, AlterViewAs, AlterViewSchemaBinding, ColumnDefinition, CreateFunction, CreateView, LogicalPlan, QualifiedColType, ReplaceColumns, V2CreateTablePlan}
+import org.apache.spark.sql.catalyst.rules.Rule
+import org.apache.spark.sql.internal.SQLConf
+import org.apache.spark.sql.types.{DataType, ImplicitStringType, StringType}
+
+object ResolveImplicitStringTypes extends Rule[LogicalPlan] {
```

Comment:
@stefankandic Not all commands have been ported on V2 API yet, so, V1 commands like `ClearCacheCommand` are in  the `sql` package. And you cannot access to them from `catalyst`, and cannot handle them.

If it is need, we could at least create a logical nodes for them, see https://issues.apache.org/jira/browse/SPARK-33392. Let me know which commands you need to handle that are not ported to DS V2 API.

Reply:
I think we just need create/alter for table/view/schema. Is that doable?